es-vue-base/src/lib-components/EsCollapse.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+    <div>
+        <EsButton
+            block
+            :aria-label="id"
+            class="collapse-holder text-left p-0 mb-3 font-weight-bold text-black d-flex align-items-center justify-content-between text-decoration-none text-body"
+            variant="link"
+            @click="isCollapsed = !isCollapsed">
+            <div>
+                <slot name="title" />
+            </div>
+            <div>
+                <IconChevronDown
```

Comment:
I wonder if we should use a `transform: rotate(90deg)` vs. hide/show two different icons. Does the hide/show trigger a layout-shift?

Reply:
Yep, added a transition feels smother good suggestion. c7e1bd3